Roadmap.md

```diff
@@ -1,70 +1,107 @@
-# Future Roadmap
-### Where we plan to strengthen docfx for all documentations
+# Roadmap
 
-- [x] Feature already available in [docfx](RELEASENOTE.md)
+For any released features, please see notes in [release notes](RELEASENOTE.md).
 
-## Near-term
+You may have noticed that the team is working on DocFX v3 right now. Yes, v3 is the long-term plan for DocFX to be the end-to-end "docs" tool to support
+
+- markdown and structured conceptuals (such as landing page, tutorial, etc.)
+- generation of reference language documentation from source codes or released packages (such as .NET, REST, Java, etc.)
+- able to run everywhere (cross-platform) and publish to anywhere possible (static site, or a built-in rendering stack similar as <https://docs.microsoft.com>)
+- look and feel defaulted to docs.microsoft.com, but still customizable
+
+The reason why we choose to have a redesign and implementation in v3 rather than in v2 is mainly due to following considerations:
+
+- The v2 plugin framework is too flexible that makes it difficult to make changes in DocFX without impacting the plugins. The flexibility also hinders community users to dig deep into the code and contribute.
+- It is not easy to local debug and test due to the existence of plugin framework.
+- The performance is also not ideal: two major issues are with AppDomain and Git related operations.
+- Technical stacks are not consistent causing additional overhead in development and troubleshooting.
+- Community users are expressing their desire to have a documentation experience similar to <https://docs.microsoft.com> (i.e. [feature requests](README.md#collecting-feedbacks-and-proposals-for-docfx) on versioning, PDF link, REST definition pages, etc.), but there is no easy way to approach these requirements in v2.
+
+## v3 timeline
+
+v3 is a long journey that requires a substantial amount of time to deliver even the basic end-to-end experience parity to v2. In order to have better management and achieve continuous delivery, we break it down into various phases.
+
+Phase 1 of v3 has been completed at the end of June 2018. At this stage, v3 is able to build markdown conceptuals, TOC, resources into an intermediate model (what we call raw page JSON model). We've also implemented the integration with <https://docs.microsoft.com> internally for Microsoft-owned content repos.
+
+Phase 2 of v3 is targeted to the end of September 2018 with the goal to support 100% parity with v2 for markdown and structured conceptuals (via SDP). We're also going to prepare necessary tooling for migration from v2 to v3.
```

Comment:
>SDP [](start = 145, length = 3)

minor: AWEs (Acronyms Without Explanations) must be explained on first use in communications

Reply:
great! will fix.